Week-3/Ex-1/index.js

```diff
@@ -0,0 +1,30 @@
+
+function memoize(fn) {
+    let cache = new Map();
+    return function(...args) {
+        const key = args.toString();
```

Comment:
What about the edge cases?

What if the args is undefined or null?

Reply:
Noted.